.gitignore

```diff
@@ -25,3 +25,5 @@ $RECYCLE.BIN/
 Thumbs.db
 UserInterfaceState.xcuserstate
 .env
+./vscode-data.json
+./src/components.d.ts
```

Comment:
this contains the components types... am I missing the reson you need to ignore it?

Reply:
np I can close this pr.. I thought that since they were auto-generated files every time you launch the server, they were not useful. The reason I wanted to ignore them was simply because they generate conflicts when switching from one branch to another..